### ISSUE TITLE
[Feature Request] Add Custom Exclusions in Configuration

### ISSUE THREAD

**Issue by cvocvo** [yawkale/Dyfort.Umbraco.AutoImageOptimze#2]
It would be awesome if you could add custom exclusions in the configuration. 
I see here (we're using v10 Umbraco for LTS so we're on v1.0.2 of the plugin):
https://github.com/yawkale/Dyfort.Umbraco.AutoImageOptimze/blob/1.0.2/Dyfort.Umbraco.AutoImageOptimze/AutoImageOptimizeComposer.cs#L63 that you're already excluding `/umbraco/assets`. It seems like the configuration could be extended like this:
```
 "AutoImageOptimizer": {
    "Quality": "92",
    "AllowedExtentions": [
      ".png",
      ".jpg",
      ".jpeg"
    ],
    "ExcludedFolderPaths": [
      "/foo/",
      "/foo/bar/",
      "/full-quality-photos-downloads/"
    ],
    "Enabled": "true"
  }
```

Thank you!

**Comment by yawkale** [comment]
fixed in 1.0.3 and 2.0.1

**Comment by cvocvo** [comment]
Thank you @yawkale!
